CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 [1]: https://www.npmjs.com/package/nodejs-logging?activeTab=versions
 
+## v3.0.1
+
+### Fixes
+- fix(deps): update dependency @google-cloud/logging to v3 (#195)
```

Comment:
Too late now, but given that this changelog is for the users of this module, this commit is poorly worded. IMO, it should be something like: `fix(samples): upgrade ...`. The way it is worded right now makes it very confusing.

Reply:
Yeah :(  I need to be more careful about these when they're filed.